website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+## Using dbt-firebolt
+
+### `quote_columns`
+
+To prevent a warning, you should add a configuration as below to your `dbt_project.yml`. For more info, see the [relevant dbt docs page](https://docs.getdbt.com/reference/resource-configs/quote_columns).
+
+
+<File name>
+```yml
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
+```
+</File>
+
+
+### Model Configuration for Fact Tables
+
+A dbt model can be created as a Firebolt fact table and configured using the following syntax:
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
+
+<TabItem value="project-yaml">
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  [<resource-path>]:
```

Comment:
Should these brackets be here?

Reply:
No - not sure how those got in because it is not displayed that way in our [readme](https://github.com/firebolt-db/dbt-firebolt).